### PR TITLE
DOC change axhspan to numpydoc format

### PR DESCRIPTION
**Pull request by maluethi**
Greetings from Rivo and Matthias at ASPP summer school.

@NelleV

### CODE REVIEW

**Comment by NelleV** [comment]
This looks good! Thanks for the patch.
Cheers,
N

**Comment by NelleV** [comment]
@tacaswell This PR is ready for review (and merge).

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -816,22 +809,33 @@ def axhspan(self, ymin, ymax, xmin=0, xmax=1, **kwargs):
...
+        xmax : float, optional, default: 1
+               Upper limit of the vertical span in axes (relative
+               0-1) units.
 
-        Valid kwargs are :class:`~matplotlib.patches.Polygon` properties:
+        Returns
+        -------
+        Polygon : `~matplotlib.patches.Polygon`
 
-        %(Polygon)s
```
Comment (QuLogic) [inline lib/matplotlib/axes/_axes.py]: why remove this?

**Comment by QuLogic** [comment]
Your commit is currently attributed to "maluethi". It is your choice to use a real name or not, of course, but I'm just making a note of it in case it was unintended.

File: lib/matplotlib/axes/_axes.py (hunk 1 above)
Comment (NelleV) [inline lib/matplotlib/axes/_axes.py]: It's been moved to the section "Other parameters".

Comment (QuLogic) [inline lib/matplotlib/axes/_axes.py]: Only the general line (i.e., `kwargs`), but not this substitution of all the keyword arguments.

Comment (maluethi) [inline lib/matplotlib/axes/_axes.py]: Unfortunately the list reference got lost during the conversion process. I fixed this in the new commit.

**Comment by NelleV** [comment]
The failing test seem unrelated so I think we are good to merge (thought it be good to relaunch the failing test on travis to make sure).
N

**Comment by QuLogic** [comment]
The two commits are attributed to two different names and two different email address, though they seem to refer to the same person; please confirm that is what you really want to do.

**Comment by maluethi** [comment]
@QuLogic: This is fine by me.

**Comment by tacaswell** [comment]
backported to v2.x as 0234978

Thanks for your work!

**Pull request merged**